gnome-initial-setup/pages/account/gis-account-page.c

```diff
@@ -258,6 +299,18 @@ on_shared_user_created (GtkWidget       *page_local,
     gis_driver_set_user_permissions (GIS_PAGE (page)->driver, user, password);
 }
 
+gboolean
+gis_account_page_is_eula_enabled (void)
```

Comment:
`gis_account_page_should_show_eula` seems a better name, since the config key is called `show-eula`.

Reply:
This function should take a `GisPage *page` and use `page->driver` to get the driver.